### DISCUSSION TITLE
Is it possible to have a default preselected position for a marker

### DISCUSSION THREAD

**Discussion by angra007** [patrykandpatrick/vico#876] [Questions]
### Question

I have a line graph with a custom marker. I want to preselect the marker in the centre of the graph which will be shown when user is not interacting with the graph. Is it possible to do this?  

### Vico version(s)

2.0.0-aplha.28

### UI framework(s)

Jetpack Compose

**Comment by angra007** [comment]
So far I am using `persistentMarker` for the default position. I need a way to hide the `persistentMarker` when the user interacts with the graph and show it back again once the user stops interaction.

**Reply by Gowsky** [reply]
Hello, @angra007. You can do this by using `CartesianMarkerVisibilityListener`. Note that there's a recomposition issue related to strong skipping mode in version `2.0.0-beta.1` (#900), and it may get in the way. It will be fixed in the next update.
